pcsx2/GS/GSCapture.h

```diff
@@ -20,6 +20,8 @@
 
 #ifdef _WIN32
 #include "Window/GSCaptureDlg.h"
+#else
+#include "fmt/core.h"
```

Comment:
just include normally, it's not going to fail to compile or anything.

Reply:
I didn't because it's not used on windows here.

Reply:
I understand. Just include it anyway.